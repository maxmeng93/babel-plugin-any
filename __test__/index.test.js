import plugin from '../src/index';
import pluginTester from 'babel-plugin-tester';

pluginTester({
  plugin: plugin,
  tests: {
    'test-CallExpression': {
      code: `
        sum(1,2); 
        sum(1,2,3)
      `,
      snapshot: true
    },
    'test-FunctionDeclaration': {
      code: `
        function sum(a, b) {
          return a + b;
        }
      `,
      snapshot: true
    }
  }
})