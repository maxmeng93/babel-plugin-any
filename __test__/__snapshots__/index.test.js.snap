// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unknown plugin test-CallExpression: test-CallExpression 1`] = `

sum(1,2); 
sum(1,2,3)

      ↓ ↓ ↓ ↓ ↓ ↓

sum([1, 2]);
sum([1, 2, 3]);


`;

exports[`unknown plugin test-FunctionDeclaration: test-FunctionDeclaration 1`] = `

function sum(a, b) {
  return a + b;
}

      ↓ ↓ ↓ ↓ ↓ ↓

function sum(nums) {
  return nums.reduce((sum, next) => sum + next, 0);
}


`;
